NEWS.md

```diff
@@ -1,3 +1,8 @@
+# nflfastR (development version)
+
+* Internal change to `calculate_player_stats()` that reflects new nflverse data infrastructure.
+* `calculate_player_stats()` now joins player information like position or headshot url via `nflreadr::load_players()`.
```

Comment:
Can you add a list of new columns here as a reference? (Makes easier to point people at when their code breaks for dupl colnames)